app/controllers/spaces_controller.rb

```diff
@@ -30,9 +30,13 @@ def create
   end
 
   def edit
+    @space = Space.find(params[:id])
```

Comment:
At a later stage, you may refactor this by placing the repeated action into a method and calling it by `before_action`